docs/conceptual-framework.md

```diff
@@ -356,4 +356,5 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
 + Results reproduced by [@nicoella](https://github.com/nicoella) on 2024-08-18 (commit [`e65dd95`](https://github.com/castorini/pyserini/commit/e65dd952d62d0eb105f24d9f45a961a6c1ad52da))
 + Results reproduced by [@natek-1](https://github.com/natek-1) on 2024-08-19 ( commit [`e65dd95`](https://github.com/castorini/pyserini/commit/e65dd952d62d0eb105f24d9f45a961a6c1ad52da))
 + Results reproduced by [@setarehbabajani](https://github.com/setarehbabajani) on 2024-08-31 (commit [`0dd5fa7`](https://github.com/castorini/pyserini/commit/0dd5fa7e94d7c275c5abd3a35acf64fbeb3013fb))
-+ Results reproduced by [@anshulsc](https://github.com/anshulsc) on 2024-09-07 (commit [`2e4fa5d`](https://github.com/castorini/pyserini/commit/2e4fa5ddc8059e0e6655b1db2591f8f069db52fd))
\ No newline at end of file
++ Results reproduced by [@anshulsc](https://github.com/anshulsc) on 2024-09-07 (commit [`2e4fa5d`](https://github.com/castorini/pyserini/commit/2e4fa5ddc8059e0e6655b1db2591f8f069db52fd))
++ Results reproduced by [@r-aya](https://github.com/r-aya) on 2024-09-08 (commit [`33ccf88`](https://github.com/castorini/pyserini/commit/33ccf88d8b173ac0c032c6af41c6583445da6ab0))
```

Comment:
When I visit this URL: https://github.com/castorini/pyserini/commit/33ccf88d8b173ac0c032c6af41c6583445da6ab0

I get the error: This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Please fix the URL to point to a commit in main trunk.